src/index.ts

```diff
@@ -229,15 +230,70 @@ const main = (vortex: VortexExtensionContext): boolean => {
         in Vortex.
       `)),
 
-      validate: wrapValidate(vortex, vortexApi, internalLoadOrderer),
-      deserializeLoadOrder: wrapDeserialize(vortex, vortexApi, internalLoadOrderer),
-      serializeLoadOrder: wrapSerialize(vortex, vortexApi, internalLoadOrderer),
-    });
+        validate: wrapValidate(vortex, vortexApi, internalLoadOrderer),
+        deserializeLoadOrder: wrapDeserialize(vortex, vortexApi, internalLoadOrderer),
+        serializeLoadOrder: wrapSerialize(vortex, vortexApi, internalLoadOrderer),
+      });
+
+    }
+
+    vortex.registerReducer(VORTEX_STORE_PATHS.settings, settingsReducer);
+
+    vortex.registerSettings(`V2077 Settings`, settingsComponent, undefined, () => {
+      const state = vortex.api.store.getState();
+      const gameMode = vortexApi.selectors.activeGameId(state);
+      return gameMode === GAME_ID;
+    }, 51);
+
+    // Auto convert TODO
```

Comment:
```suggestion
    // 0.9.0 information TODO
```
my bad. copied and pasted without fully reviewing the changes.